solr/core/src/java/org/apache/solr/security/JWTVerificationkeyResolver.java

```diff
@@ -106,7 +106,7 @@ public Key resolveKey(JsonWebSignature jws, List<JsonWebStructure> nestingContex
         }
       }
 
-      // Add all keys into a master list
+      // Add all keys into a primary list
```

Comment:
This occurrence of "master" is unrelated to master/slave replication. Maybe simply remove the word "master" or replace it with "reference" but "primary" doesn't really make sense.

Reply:
yea, that was a screw up of mine. I would say this can stay master as I have no inherent problem with the word `master` outside of the context of `master/slave`.

However, I agree that probably best to remove this one in this case because others do.  I'll change it to reference.